app/javascript/src/Stats/inlinechart/index.jsx

```diff
@@ -0,0 +1,137 @@
+// @flow
+
+type Props = {
+    endPoint: string,
+    metricName: string,
+    title: string,
+    unitPluralized: string
+}
+
+type State = {
+    loading: boolean,
+    title: string,
+    total: string,
+    values: Array<number>,
+    unit: string,
+    unitPluralized: string
+}
+
+type RefObject = { current: null | HTMLDivElement }
+
+type DataMetric = {
+    unit: string
+}
+
+type Data = {
+    metric: DataMetric,
+    total: number,
+    values: Array<number>
+}
+
+type ResponseError = {
+  status: string,
+  statusText: string
+}
+
+import React, {Component} from 'react'
+import c3 from 'c3'
+import 'whatwg-fetch'
+import 'core-js/es6/promise'
+import 'url-polyfill'
+import moment from 'moment'
+
+class InlineChart extends Component<Props, State> {
+  c3ChartContainer: RefObject
+
+  constructor (props: Props) {
+    super(props)
+    this.c3ChartContainer = React.createRef()
+    this.state = {
+      loading: true,
+      title: this.props.title,
+      total: '',
+      values: [],
+      unit: '',
+      unitPluralized: this.props.unitPluralized
+    }
+  }
+
+  generateC3Chart () {
+    const nodeElem = this.c3ChartContainer.current
+    c3.generate({
+      bindto: nodeElem,
+      axis: {
+        x: { show: false },
+        y: { show: false }
+      },
+      legend: { show: false },
+      point: { show: false },
+      data: {
+        columns: [
+          [...this.state.values]
+        ]
+      },
+      tooltip: {
+        contents: function (d) {
+          return `<span><i class='tooltip-dot'></i> ${d[0].value}</span>`
+        }
+      },
+      onresize: function () {
+        nodeElem.style.maxHeight = 'none'
+      }
+    })
+  }
+
+  getTotalAsString (total: number, unit: string) {
+    return `${total.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')} ${unit.substring(0, 10)}`
```

Comment:
Since we are already using Numeral, you could import it here and do things like: https://github.com/3scale/porta/blob/master/app/javascript/src/Stats/lib/chart.jsx#L34